lib/fungus_toast/games/games.ex

```diff
@@ -11,6 +11,8 @@ defmodule FungusToast.Games do
   alias FungusToast.Games.{Game, GameState, Grid, Round, GrowthCycle, MutationPointsEarned}
 
   @starting_mutation_points 5
+  @starting_end_of_game_count_down 5
+  def starting_end_of_game_count_down, do: @starting_end_of_game_count_down
```

Comment:
Why declare a module attribute just to wrap it in a function?

Reply:
You can't access module attributes outside of the module (as I understand): https://stackoverflow.com/questions/37713244/access-module-attributes-outside-the-module